tools/run_tests/dockerize/build_and_run_docker.sh

```diff
@@ -57,6 +57,7 @@ CONTAINER_NAME="build_and_run_docker_$(uuidgen)"
 docker run \
   "$@" \
   --cap-add SYS_PTRACE \
+  --cap-add NET_ADMIN \
```

Comment:
I don't like the idea of adding a new capability for all X thousand tests when there's just one that needs this.

Reply:
What's a better way to do this that would only result in this one test running with NET_ADMIN capability?